applications/luci-app-aria2/root/usr/libexec/aria2-call

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+action=$1
+shift
+
+[ -r "$1" ] || exit 1
+
+case "$action" in
+	cat)
+		cat $1
```

Comment:
Instead of allowing arbitrary files here you should pull these values out of the uci configuration. This way a read only user may not be able to view resources other than the ones configured.

Reply:
Done